README.md

```diff
@@ -45,7 +45,7 @@ For other options (tarballs, debs, etc.), see
 ## Documentation and Usage
 
 For in-depth instructions on installation and building the
-documentation, see the [SymPy Documentation Style Guide
+documentation, see the SymPy Documentation Style Guide
```

Comment:
I think that the intention was to have a closing square bracket at the end of the line as well so that it renders with this as the text of the link rather than showing the URL.

Reply:
```suggestion
documentation, see the [SymPy Documentation Style Guide]
```